### ISSUE TITLE
Focus indicator doesn't leave empty monitor

### ISSUE THREAD

**Issue by dalyIsaac** [dalyIsaac/Whim#153]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

When a window is removed from a monitor, and the monitor doesn't have any other windows, the focus indicator doesn't disappear.

### Expected Behavior

When the last window is removed from a monitor, the focus indicator disappears.

### Steps To Reproduce

1. Open Notepad
2. Move to its own monitor
3. Switch the monitor to another workspace

### Environment

```markdown
- Windows: 22000.778
- Architecture: x64
- Whim: v0.1.5-alpha+8dc63b84
```
